test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -77,3 +77,8 @@ def test_resample(self, orig_freq, new_freq):
         transform = T.Resample(orig_freq=orig_freq, new_freq=new_freq)
         waveform = get_whitenoise(sample_rate=8000, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform])
+
+    def test_fade(self):
```

Comment:
Can you parameterize `fade_shape` parameter? 

https://github.com/pytorch/audio/blob/90311ed11ff88042b8ea5e849a746aedb10a0f42/torchaudio/transforms.py#L797-L810

Reply:
Sure. WIll do!.